client/utils/account-fees.tsx

```diff
@@ -302,6 +319,17 @@ export const formatMethodFeesDescription = (
 	} );
 };
 
+export const formatPillFeesDescription = (
```

Comment:
Following my previous comments, I think we could spare this function and stick to the original implementation.

Reply:
This function is now redundant and has been deleted, since we are no longer wanting to show a separate rate on the Pill . We will show the same discounted rate.